artifactory.py

```diff
@@ -2585,15 +2585,16 @@ def get_build_info(self, build_name, build_number):
         return self._get_info(build_name, build_number)
 
     def _get_info(self, build_name, build_number=""):
-        url = build_name
+        url = requests.utils.quote(build_name, safe="")
```

Comment:
Does it encode `/` right? 
In the related issue #366 the vars would be"
```python
build_name = "maven-demo"
build_number = "1-build-snapshot"
```
but here we're encoding only "maven-demo" part and then we add a slash, so the url at the end is `maven-demo/1-build-snapshot` and not `maven-demo%2F1-build-snapshot` as it supposed to be

Reply:
I have tested it, it encode `/`  right.

Reply:
The correct build_name is `maven-demo/1-build-snapshot`
If we don't encode the build_name.
`maven-demo` will be treated as build_ name.
To slove the problem,  build_name should be encoded.
requests.utils.quote(build_name, safe="") can encoded `maven-demo/1-build-snapshot` to `maven-demo%2F1-build-snapshot`.

Reply:
I see, thank you for the explanation!